navbar/navbar.css

```diff
@@ -0,0 +1,18 @@
+@import url(/style.css);
+
+.nav-bar{
+    display: flex;
```

Comment:
you can put flex in utility class if it used frequently

Reply:
sure! will see in all the components and update them all together.